frontend/src/api/models.ts

```diff
@@ -19,12 +19,16 @@ export type ChatRequestOverrides = {
     vectorSearch?: boolean;
 };
 
-interface DialogRequest {
+export interface DialogRequest {
     userID: string;
     conversationID: string;
     dialogID: string;
 }
 
+export interface RateRequest extends DialogRequest {
+    rating?: boolean | undefined;
```

Comment:
From a boolean perspective do we need to set undefined? Both are falsy.

Reply:
Yup! Rating being set to undefined has meaning to the user here. Instead of setting this value to true or false, we use undefined to mean that both thumbs-up and thumbs-down are unset.